.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  push:
+    tags:
```

Comment:
you might want to limit the branch of those tags...(i.e. to master)

Reply:
Good point,
Surprisingly, github doesn't provides a way to filter tag + branch,

The following is processed as `this OR this` which means the pipeline will trigger in every merge to the main branch and in every new tag regardless of the target branch

```
on:
  push:
    branches:
      - main
    tags:
      - 'v**'
```

Filtering by `github` context can't help here since the context for tag event has nothing about the target branch,

There is a workaround [0] checking if the new tag exists on <branch> but it complexes a bit the pipeline,
IMO, since no one can push tags (to any origin branch) except the maintainers, we can leave it that way for now, WDYT?

[0] https://github.com/rickstaa/action-contains-tag/#use-case

Reply:
Yeah,

even in python-driver, I don't limit it, just checking the tag:
https://github.com/scylladb/python-driver/blob/master/.github/workflows/build-push.yml#L143

also I've seen this repo has unittests, it might be a good thing to make sure they are running and passing prior to releasing (and also on PRs)

Reply:
> even in python-driver, I don't limit it, just checking the tag:
> https://github.com/scylladb/python-driver/blob/master/.github/workflows/build-push.yml#L143

I see the `endsWith(github.event.ref, 'scylla')` trick

> also I've seen this repo has unittests, it might be a good thing to make sure they are running and passing prior to releasing (and also on PRs)

This PR is about release but yes, we need one for CI, there is an issue (and a [draft PR](https://github.com/scylladb/scylla-cli/pull/34/files)) I should take care of when I'll have time...

If you have no more comments let's approve and merge.